projects/cdk/tokens/is-ios.ts

```diff
@@ -1,8 +1,12 @@
 import {inject, InjectionToken} from '@angular/core';
-import {USER_AGENT} from '@ng-web-apis/common';
+import {NAVIGATOR, USER_AGENT} from '@ng-web-apis/common';
 
 const IOS_REG_EXP = /ipad|iphone|ipod/;
+const APPLE_REG_EXP = /apple/;
 
 export const TUI_IS_IOS = new InjectionToken<boolean>('iOS browser detection', {
-    factory: () => IOS_REG_EXP.test(inject(USER_AGENT).toLowerCase()),
+    factory: () =>
+        IOS_REG_EXP.test(inject(USER_AGENT).toLowerCase()) ||
+        (APPLE_REG_EXP.test(inject(USER_AGENT).toLowerCase()) &&
+            inject(NAVIGATOR).maxTouchPoints > 1),
```

Comment:
can we add tests for this?

Reply:
I thought about test, but i can't find tests for tokens
Can i add subfolder "tokens/tests" ?
<img width="446" alt="Screenshot 2022-02-25 at 21 45 53" src="https://user-images.githubusercontent.com/5115677/155771241-0daf40a7-3183-456d-a67c-c25475b19aa1.png">

Reply:
yes

Reply:
add tests